<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>209</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Capital Letters</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="Special_Characters">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Special Characters</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add No Repeating Characters</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Numbers</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Lower Case Letters</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="generateButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Generate Password</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password Length:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPassword_Generator">
    <property name="title">
     <string>Don't Tell Anyone</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuPassword_Generator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
